.github/workflows/autorelease.yaml

```diff
@@ -0,0 +1,24 @@
+name: Autorelease
+on:
+  push:
+    branches:
+      - main
+      - release-automation
```

Comment:
Will be removed after review. This is only to test job. 
Job runs right now and created valid release.
```suggestion
```